packages/terra-graphs-docs/src/example-datasets/dataObjects/Combination/lineSimpleRegionDataset.js

```diff
@@ -0,0 +1,28 @@
+import Carbon from '@cerner/carbon-graphs/lib/js/carbon';
+import {
+  loadPopup,
```

Comment:
nit: remove **,** to make it a single line

Reply:
Fixed [here](https://github.com/cerner/terra-graphs/pull/235/commits/2693d1695999667f0eaee02713222567d35784ed).